src/block/progress-circle/save.js

```diff
@@ -0,0 +1,62 @@
+import ProgressCircleStyles from './style'
+import {
+	BlockDiv,
+	CustomCSS,
+	getResponsiveClasses,
+} from '~stackable/block-components'
+import { version as VERSION } from 'stackable'
+import { withVersion } from '~stackable/higher-order'
+import classnames from 'classnames'
+
+import { InnerBlocks } from '@wordpress/block-editor'
+import { compose } from '@wordpress/compose'
+
+const Save = props => {
+	const { className, attributes } = props
+	const responsiveClass = getResponsiveClasses( attributes )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-progress-circle',
+		responsiveClass,
+	] )
+
+	const divClassNames = classnames( [
+		'stk-progress-circle',
+		{
+			'with-animation': attributes.progressAnimate,
+		},
+	] )
+
+	return (
+		<BlockDiv.Content
+			className={ blockClassNames }
+			attributes={ attributes }
+		>
+			<ProgressCircleStyles.Content { ...props } />
+			<CustomCSS.Content attributes={ attributes } />
+			<div
+				className={ divClassNames }
+				role="progressbar"
+				aria-valuemin="0"
+				aria-valuemax="100"
+				aria-valuenow={ attributes.progressPercent }
+				aria-valuetext={ attributes.progressAriaValueText }
```

Comment:
As a precautionary measure to prevent user input from messing up the block structure, please pass the text value through `striptags()`